src/namespaces/genai/genAIIndex.ts

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Any reason for being genAIIndex?

For consistency, I think we can call this `genai.ts`